master/internal/usergroup/group_intg_test.go

```diff
@@ -84,13 +84,23 @@ func TestUserGroups(t *testing.T) {
 
 	t.Run("search groups by user membership", func(t *testing.T) {
 		groups, _, count, err := SearchGroupsWithoutPersonalGroups(ctx, "", testUser.ID, 0, 0)
-		require.NoError(t, err, "failed to search for groups that user blongs to")
+		require.NoError(t, err, "failed to search for groups that user belongs to")
 
 		index := groupsContain(groups, testGroup.ID)
 		require.Equal(t, 2, count, "group search returned wrong count")
 		require.NotEqual(t, -1, index, "Group user was added to not found when searching by user membership")
 	})
 
+	t.Run("manually edit groups query", func(t *testing.T) {
+		query := SearchGroupsQuery("", testUser.ID, false)
+		query = query.Where("group_name = ?", testGroup.Name)
+		groups, _, count, err := SearchGroupsPaginated(ctx, query, 0, 0)
+		require.NoError(t, err, "failed to search for group in modified query")
+		index := groupsContain(groups, testGroup.ID)
```

Comment:
Nit: since this isn't used until line 101, I'd move it down to immediately before that for readability. A blank line between the two might additionally help, but that seems even less important.